### ISSUE TITLE
[nightly] Android Touch Problem.

### ISSUE THREAD

**Issue by ikusorae** [renpy/renpy#2633]
The game cannot be manipulated because it is not touchable on Android.
I created and tested a new project, so I don't think it's a problem with a specific project.

(Also it seems to occur in certain situations in iOS, but it is not clear.)

**Comment by renpytom** [comment]
Which phone is this?

**Comment by ikusorae** [comment]
Galaxy Note10 with Android 11 Beta.

**Comment by renpytom** [comment]
Do you have the model number (should be in settings). I'm considering getting a used one in for testing.

**Comment by ikusorae** [comment]
The model number is SM-N971N. I haven't tested it yet, but other Android 11-based Samsung phones may have similar problems. Or maybe it's a matter of a particular language. (This is also unclear)
I'm going to also test it through another device today. Thanks.

**Comment by renpytom** [comment]
If you can do that, I'd appreciate it.  The SM-971N is $500 used, so if you can find another phone that it's broken on, that would help. (It works fine on my pixel 5, 2xl, and Galaxy S5.)

**Comment by ikusorae** [comment]
I tested it on other Samsung devices such as the Galaxy S10 5G (SM-G977N) with Android 10 and the Galaxy Note 20 Ultra (SM-N986N) with Android 11.
As a result, the two devices worked perfectly.
What I can look forward to in the future is that Android 11 on my Note 10 is a Beta version, so it can be modified with additional updates from Samsung. (Because it worked well with Note 20 with stable version of Android 11 installed.)
It could be a problem that only occurs on my device among Note10, so I'm going to test it by finding Note10 users like me in places like forums. I will then continue to share the information. Thank you.

**Comment by ikusorae** [comment]
Also, apart from the above issues, moving the scrollbar in my project causes the game to stop on all platforms (macOS, iOS, Windows…).
There is no problem in 7.3.5 and then there is a problem from 7.4.0, so could you check it if I send my project by email?

**Comment by renpytom** [comment]
Please do.

**Comment by ikusorae** [comment]
I sent you an email. Please check.
Thank you.

**Comment by ikusorae** [comment]
Yep. I've tested it in many ways and found the reason for the problem that touching doesn't work.  
When activate the magnification in accessibility, the problem occurs. This is not the problem of using the function, but just the matter of activating it. 
When I disactivate the function, the game works normally. 
The former version didn't make this kind of problems so I assume that new version of Ren'py just makes some crashes. 
But there is still the problem that related to scroll bar I mentioned in previous email remains regardless of platforms. 
If you find the solution for the problem that I mentioned in previous email, I wish you to write back on the matter. 

Thanks.

**Comment by renpytom** [comment]
I've figured out the problem with the scrollbars, and the fix will be in the 7.4.2 release. (It's not done yet, since I need to test potential fixes.) 

Do you have the ability to try other samsung phones to see if they exhibit the problem? I'm especially wondering about the S10e, which seems much more affordable. (But I don't want to get one if it doesn't exhibit the problem.)

**Comment by ikusorae** [comment]
I do not have S10e now, so tried in another way to test the problem. 
In Galaxy S20(SM-G981N) with Android 10, touching works well regardless of activating magnification function. 
I tested on the same device with upgraded OS, Android 11, the touching problem occurred again. 
It seems Samsung phones with Android 11 have this kind of problem. 
I assume S10e with Android 11 may have the same problem. (Samsung's update date may differ from the location, countries, or models. So some S10e devices may not updated to Android 11.)

Thanks.

**Comment by renpytom** [comment]
Okay, just ordered an S10e, which looks like it has Android 11 in the US. We'll see what it's like when it gets here.

**Comment by renpytom** [comment]
Finally got the one UI 3.0 update, and tried this, and it worked. Could you try a nightly, and see if it's fixed for you?

**Comment by ikusorae** [comment]
Yes, I think that the issue has been resolved in recent versions of nightly. Thank you.
